>>> 'Pkg.add("FaceDatasets")' log
INFO: Installing Blosc v0.1.4
INFO: Installing ColorTypes v0.1.3
INFO: Installing ColorVectorSpace v0.0.2
INFO: Installing Colors v0.5.2
INFO: Installing Docile v0.5.16
INFO: Installing FaceDatasets v0.1.2
INFO: Installing FixedPointNumbers v0.0.10
INFO: Installing Graphics v0.1.3
INFO: Installing HDF5 v0.5.5
INFO: Installing Images v0.4.46
INFO: Installing MAT v0.2.12
INFO: Installing Reexport v0.0.2
INFO: Installing SIUnits v0.0.5
INFO: Installing TexExtensions v0.0.2
INFO: Installing Zlib v0.1.8
INFO: Building Blosc
INFO: Building HDF5
INFO: Building Images
INFO: Package database updated

>>> 'Pkg.test("FaceDatasets")' log
Julia Version 0.4.0-pre+7097
Commit c4b089f (2015-08-29 13:17 UTC)
Platform Info:
  System: Linux (x86_64-unknown-linux-gnu)
  CPU: Intel(R) Core(TM) i5-2500K CPU @ 3.30GHz
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Sandybridge)
  LAPACK: libopenblas
  LIBM: libopenlibm
  LLVM: libLLVM-3.3
INFO: Testing FaceDatasets
INFO: Recompiling stale cache file /home/vagrant/.julia/lib/v0.4/Images.ji for module Images.
INFO: Recompiling stale cache file /home/vagrant/.julia/lib/v0.4/HDF5.ji for module HDF5.
ERROR: LoadError: LoadError: LoadError: UndefVarError: Gray not defined
 in include at ./boot.jl:259
 in include_from_node1 at ./loading.jl:271
 in include at ./boot.jl:259
 in include_from_node1 at ./loading.jl:271
 in require at ./loading.jl:210
 in include at ./boot.jl:259
 in include_from_node1 at ./loading.jl:271
 in process_options at ./client.jl:308
 in _start at ./client.jl:411
while loading /home/vagrant/.julia/v0.4/FaceDatasets/src/core.jl, in expression starting on line 5
while loading /home/vagrant/.julia/v0.4/FaceDatasets/src/FaceDatasets.jl, in expression starting on line 6
while loading /home/vagrant/.julia/v0.4/FaceDatasets/test/runtests.jl, in expression starting on line 1
============================[ ERROR: FaceDatasets ]=============================

failed process: Process(`/home/vagrant/julia/bin/julia --check-bounds=yes --code-coverage=none --color=no /home/vagrant/.julia/v0.4/FaceDatasets/test/runtests.jl`, ProcessExited(1)) [1]

================================================================================
ERROR: FaceDatasets had test errors
 in error at ./error.jl:21
 in test at pkg/entry.jl:753
 in anonymous at pkg/dir.jl:31
 in cd at file.jl:22
 in cd at pkg/dir.jl:31
 in test at pkg.jl:71
 in process_options at ./client.jl:284
 in _start at ./client.jl:411

>>> End of log
